### ISSUE TITLE
Send Just One Hotbar

### ISSUE THREAD

**Issue by brunocalado** [Norc/player-configurator#2]
Hi. This is a great module!!!

Can you make possible send just one macro bar to the players?

I have 2 hotbars. One for gming and the other one for players. I just want to send the player hotbar.

**Comment by BronzeToad** [comment]
I'm afraid to use this because I wasn't sure how the modules behaves. Does it send all five pages of your macro hotbar?

**Comment by Norc** [comment]
Yeah, it does send all currently. This is a good suggestion, thanks.

I plan to release an improved version of this in the next month or so, just doing a version compat update for now since it should be working fine in the new version.

**Comment by Armandeus66** [comment]
It would be helpful if the bar(s) the players receive is marked somehow (color?) or locked from editing, so they won't get upset when their arrangement of that bar gets overwritten by the GM if the GM wants to add to or update it later.

**Comment by danielmacapa** [comment]
any news?

**Comment by Norc** [comment]
Tonight is slotted for v10 updates for Custom Hotbar and Configurator if I
get to it. SoonTM (FinallyTM).
